docs/DeveloperGuide.md

```diff
@@ -477,9 +544,14 @@ truth value of the operation will result in 1 (TRUE) if either x == 1 or y == 1,
 * *Sopln()* - Abbreviation for java out operation "System.out.println()", from package java.lang.
 * *Standard I/O Operation* - Common java I/O streams include System.in, System.out and System.err.
 * *XNOR* - An inverse of the XOR operation. Outputs the opposite truth value of what a XOR gate would output.
-* *XOR* - Also known as exclusive OR, XOR is a secondary operation in boolean algebra which may be denoted as x XOR y. The truth value of the operation will result in 1 (TRUE) if only one of x == 1 or y == 1, and 0  for other combinations of values.
+<<<<<<< HEAD
```

Comment:
The glossary is in alphabetical , so might want to add the terms to the respective positions in the list. Moreover, remove the head >>>> merge conflict markers